plugins/wacom-i2c/wacom-i2c.quirk

```diff
@@ -0,0 +1,28 @@
+# AES bootloader mode
+[DeviceInstanceId=HIDRAW\VEN_056A&DEV_0094]
+Plugin = wacom_i2c
+Guid = WacomAES
+Flags = is-bootloader
+
+# EMR bootloader mode
+[DeviceInstanceId=HIDRAW\VEN_056A&DEV_012B]
+Plugin = wacom_i2c
+Guid = WacomEMR
+Flags = is-bootloader
+
+# Dell XPS-15
+[DeviceInstanceId=HIDRAW\VEN_056A&DEV_4875]
+Plugin = wacom_i2c
+Guid = WacomAES
+
+[Guid=WacomEMR]
+WacomI2cAddr=0x09
+WacomI2cFlashBlockSize=64
+WacomI2cFlashBaseAddr=0x2000
+WacomI2cFlashSize=0x1e000
```

Comment:
So all these quirk values are obviously no longer needed switching to hidraw (Yay!)

Reply:
Er at lest hte I2cAddr isn't, I guess I don't know if you'll be able to read the base address and block size from hidraw.  I wouldn't be surprised if it was encoded in the header or something.

Reply:
FWIW, @hughsie made the switch to hidraw today and tested the code on the XPS 15 2-in-1 you sent me recently. And the enumeration works now. So I2cAddr is not used in this device, so I guess it will be skipped.

As long as the whole FW upgrade happens in hidraw, you don't need low level information like address, buffer sizes, hidraw will take care of that for you :)

Reply:
Actually I might be wrong, you might need the block size, base addr and flash size. I do not expect the address to be required though.

Reply:
I've removed the address -- good catch.